CONTRIBUTING.md

```diff
@@ -40,14 +39,193 @@ If you happen to have a fix for a package, please have a look at the
 issue tracker to see if it doesn't concern an existing issue. If that
 is the case mention it in your pull request.
 
-
 Adding new packages
 -------------------
 
 Information about creating new packages and adding them to repository
 is available in
 [opam's manual](https://opam.ocaml.org/doc/Packaging.html).
 
+The following is a tutorial on the steps to be followed to make your
+package available in opam.ocaml.org:
```

Comment:
```suggestion
package available in [opam.ocaml.org](https://opam.ocam.org) using `dune` as the build tool and `dune-release` as the release tool.
If your workflow does not include dune, you may try using [opam-publish](...)
```
I think we should mention that this is using a specific workflow and there are other options.

Reply:
I think dune-release only supports github, not sure if it is the same for opam-publish. This may also need to be mentioned

Reply:
The objective here is to provide a `tutorial` to take the developer from point A to point B, in this case, a step-by-step procedure to get their packages submitted to opam.ocaml.org. The opam-publish instructions are already available at https://ocaml-explore.netlify.app/workflows/publishing-a-new-package-on-opam/#for-dune-users. The intent here is not to give the developer a plethora of options, but, to give them a guided approach to packaging and publishing OCaml sources.

Reply:
I understand your perspective and I think it is good to add this.
What I wanted to say is that I think we should make extra clear that users that want an automated but less opinionated tool can also look to opam-publish, for example adding a link to the opam-publish tutorial that you just mentioned.